.github/pull_request_template.md

```diff
@@ -1,5 +1,11 @@
+**Describe your changes**
+Briefly explain what you did and provide context for a clearer understanding.
+
```

Comment:
this feels like it's doubling up, it's kind of a mix between what the commit messages should be and what information should be in the changelog, maybe we can utilise those 2 spaces better rather then adding an extra thing?  
My suggestion would be add a checklist item with 'all commits briefly describe changes made' (maybe with ref to our git guidelines? 
And maybe either on the checklist item for the changelog or in the changlog readme add a line to include the context of the change.
I'm just not sure of PR descriptions are seen after merge, whereas these things are important and should be easily viewable in either the git history or release notes which are generated from changelog

Reply:
I'm also in favor of better commit messages. As for the changelog the messages can be improved but they are more of a general overview of what changed (also for other people wanting to use the software) and not a place for implementation details. Those information should either go in the commit messages or in a doc imo.